content/updates/2021-02/1-Mass-delete-your-assets.md

```diff
@@ -0,0 +1,18 @@
+---
+pim_announcement_img: ./img/asset-mass-delete.png
+pim_announcement_alt_img: "Asset mass delete screen"
+pim_announcement_audience:
+- EE
+---
+
+# Mass delete your assets
+::: meta-data type="New" features="Productivity" available="early February" link-to-doc="../articles/work-on-your-assets.html#delete-a-selection-of-assets"
+
+You can now delete several assets with just a few clicks. No doubt it will save you a tremendous amount of time!
```

Comment:
```suggestion
If you need to delete several assets from the same family at once, it's now possible to do it in the PIM user interface. No doubt it will save you a tremendous amount of time!
```